core/src/main/java/org/apache/calcite/rel/rules/MinusToAntiJoinRule.java

```diff
@@ -36,33 +36,48 @@
 /**
  * Planner rule that translates a {@link Minus}
  * to a series of {@link org.apache.calcite.rel.core.Join} that type is
- * {@link JoinRelType#ANTI}. This rule supports 2-way Minus conversion,
+ * {@link JoinRelType#ANTI}. This rule supports n-way Minus conversion,
  * as this rule can be repeatedly applied during query optimization to
  * refine the plan.
  *
  * <h2>Example</h2>
  *
+ * <p>Original sql:
```

Comment:
frankly, I think both examples are good, I would keep both.
the simple one is easier to read.

Reply:
Done!